spec/forms/upload_form_spec.rb

```diff
@@ -90,6 +90,20 @@
       end
     end
 
+    context 'given valid attributes with massive list of keywords' do
+      it 'runs search keyword jobs with random delay' do
+        upload_form = described_class.new(
+          search_engine: 'google',
+          csv_file: fixture_file_upload('upload_valid_massive_keywords.csv', 'text/csv')
+        )
+
+        allow_any_instance_of(described_class).to receive(:generate_delay_in_seconds).and_return(5) # rubocop: disable RSpec/AnyInstance
```

Comment:
🚀 
```suggestion
        allow(upload_form).to receive(:generate_delay_in_seconds).and_return(5)

```

Reply:
`allow_any_instance_of` is a massive code smell., never use it unless there is no other way 🙈